api/v1alpha1/directoryservice_types.go

```diff
@@ -99,6 +97,16 @@ type DirectoryPasswords struct {
 	Create bool `json:"create,omitempty"`
 }
 
+// DirectoryCertificates required for operation of the directory server
+type DirectoryCertificates struct {
+	// +kubebuilder:default:=ds-master-keypair
+	MasterSecretName string `json:"masterSecretName"`
+	// +kubebuilder:default:=ds-ssl-keypair
```

Comment:
I don't think we should have the defaults here. I think it's best to allow it to fail during `kubectl apply` if these fields are missing.